flintrock/ec2.py

```diff
@@ -373,6 +373,12 @@ def get_or_create_ec2_security_groups(
             from_port=7077,
             to_port=7077,
             cidr_ip=flintrock_client_cidr,
+            src_group=None),
+        SecurityGroupRule(
```

Comment:
Could you add a brief comment here so that down the line we remember what this port is for?

```python
# Spark REST Server
SecurityGroupRule(
    ...
```